watcher/kubernetes/pods.go

```diff
@@ -231,3 +264,65 @@ func (pm *PodsManager) watchEvents(ctx context.Context, lg log.Entry, registryDa
 	}()
 
 }
+
+// getPodLogFilePath returns the container file path
+func (pm *PodsManager) getPodLogFilePath(applyID, podName string, containerName string) string {
+	return fmt.Sprintf("%s/%s/%s/%s.log", pm.absoluteLogsPodPath, applyID, podName, containerName)
+}
+
+// podLogs open a container logs steam for getting the STDOUT of container
+func (pm *PodsManager) podLogs(ctx context.Context, lg log.Entry, absoluteFilePath, namespace, podName, containerName string) {
+
+	lgContainer := lg.WithFields(log.Fields{
+		"container_name": containerName,
+		"file_path":      absoluteFilePath,
+	})
+
+	go func(ctx context.Context, lg log.Entry, absoluteFilePath, namespace, podName, containerName string) {
+		file, err := os.OpenFile(fmt.Sprintf("%s", absoluteFilePath), os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
```

Comment:
Why do you need all of these permissions? isn't it for reading only?

Reply:
The watcher needs to open read and write the pod logs into a file. 
the API component should only read to file content (see api/kubernetes/routes.go)